Settings.cs

```diff
@@ -6,10 +6,10 @@ namespace PyrusApiClient
 {
 	public class Settings
 	{
-		public Settings()
+		public Settings(string origin, string filesOrigin)
```

Comment:
This breaks backward compatibility
Let's do it like this
```c#
public const string PyrusOrigin = "https://api.pyrus.com/v4";
public const string PyrusFilesOrigin = "https://api.pyrus.com/v4";

public Settings()
	: this(PyrusOrigin, PyrusFilesOrigin)
{
}

public Settings(string origin, string filesOrigin)
{
	Origin = origin;
	FilesOrigin = filesOrigin;
	RetryCount = 2;
}
```

Reply:
done